### DISCUSSION TITLE
How do I specify output files in the top level directory?

### DISCUSSION THREAD

**Discussion by heartical** [moonrepo/moon#1751] [Q&A]
My codebase assumes there is one shared artifact folder as a single deployment point.
What am I doing wrong?

tasks:
  compile:
    platform: 'system'
    inputs:
      - '@group(configs)'
      - '@group(sources)'
    command: "go build -o ../artifacts/identity ./*.go"
    outputs: 
      - '../artifacts'

**Comment by milesj** [comment]
You can't traverse upwards with an output, but you can define outputs relative from the workspace root: https://moonrepo.dev/docs/concepts/file-pattern#workspace-relative
